### ISSUE TITLE
Train the expert lip sync discriminator problem

### ISSUE THREAD

**Issue by wowopc** [Rudrabha/Wav2Lip#240]
I added 96 * 2 neural network to train mouth recognition experts

I've been training for four days. Training has been at 0.69, can't drop. This is a two-hour video of 700 people

Is there something wrong with my data set? It can't converge all the time
![image](https://user-images.githubusercontent.com/23237930/113092370-71c0c680-9220-11eb-921b-3332aa3d2187.png)

**Comment by prajwalkr** [comment]
SyncNet takes a lot of time to converge. If you are using custom datasets, ensure you have sync corrected it. Also, increase the number of iterations per epoch.

**Comment by pengfeidip** [comment]
@prajwalkr  Can you share your **training loss** ,   **training time** on dataset LRS2 train set ? 
and  which GPU device ?
